desktop/core/src/desktop/settings.py

```diff
@@ -119,6 +119,7 @@
 # Examples: "http://media.lawrence.com/media/", "http://example.com/media/"
 MEDIA_ROOT = os.path.join(BASE_DIR, 'media')
 MEDIA_URL = '/media/'
+DATA_UPLOAD_MAX_MEMORY_SIZE = 100000000
```

Comment:
as for the big files, Django was showing "request data is too big" to overcome that problem I am increasing the default value to 100MB.